python/lsst/dax/apdb/tests/_apdb.py

```diff
@@ -303,17 +297,15 @@ def test_empty_gets_0months(self) -> None:
 
         # test if a visit has objects/sources
         if self.allow_visit_query:
-            res = apdb.containsVisitDetector(visit=0, detector=0)
+            res = apdb.containsVisitDetector(visit=1, detector=1)
+            self.assertFalse(res)
+            # non-existent image
```

Comment:
`visit=1, detector=1` image is also non-existent in this test (tables are not populated). Maybe move this comment above and say "Database is empty, no images exist"